internal/content_update_policy/default_content_update_policy_resource_test.go

```diff
@@ -0,0 +1,235 @@
+package contentupdatepolicy_test
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/crowdstrike/terraform-provider-crowdstrike/internal/acctest"
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+)
+
+const defaultResourceName = "crowdstrike_default_content_update_policy.default"
+
+func TestAccDefaultContentUpdatePolicyResourceWindows(t *testing.T) {
```

Comment:
Use table driven tests. The tests in content update policy is a great example of what they should look like.